mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMavericksViewModel.kt

```diff
@@ -225,6 +222,14 @@ abstract class BaseMavericksViewModel<S : MvRxState>(
         dispatcher: CoroutineDispatcher = Dispatchers.Main.immediate,
         reducer: S.(Async<T>) -> S
     ): Job {
+        val blockExecutions = config.onExecute(this@BaseMavericksViewModel)
+        if (blockExecutions != MavericksViewModelConfig.BlockExecutions.No) {
+            if (blockExecutions == MavericksViewModelConfig.BlockExecutions.WithLoading) {
+                setState { reducer(Loading()) }
+            }
+            return viewModelScope.launch { }
```

Comment:
Should this `delay(Long.MAX_VALUE)` It could be misleading for the blocked job to complete. Suspending forever seems more accurate